src/coreclr/jit/gentree.h

```diff
@@ -8329,12 +8213,8 @@ inline regNumber GenTree::GetRegByIndex(int regIndex) const
         return AsPutArgSplit()->GetRegNumByIdx(regIndex);
     }
 #endif
-<<<<<<< HEAD
-#if !defined(TARGET_64BIT) && !defined(TARGET_WASM32) && !defined(TARGET_WASM64)
-=======
 
-#if !defined(TARGET_64BIT)
->>>>>>> 1a31bf638c6c220d20ef65f43a07a9ac562d92d9
+#if !defined(TARGET_64BIT) && !defined(TARGET_WASM32) && !defined(TARGET_WASM64)
```

Comment:
Why is `!defined(TARGET_WASM32) && !defined(TARGET_WASM64)` needed? This is already under `#if FEATURE_MULTIREG_RET` which is `0` for WASM.

Reply:
removed